mailbag/helper.py

```diff
@@ -76,6 +83,40 @@ def moveWithDirectoryStructure(dry_run, mainPath, mailbag_name, input, emailFold
 
         return file_new_path
 
+
 def saveAttachments(part):
-    return (part.get_filename(),part.get_payload(decode=True))
-    
\ No newline at end of file
+    return (part.get_filename(), part.get_payload(decode=True))
+
+
+def saveFile(filePath, text):
```

Comment:
Ideally all this will be run from a warc.py derivatives file so it can work as a plugin. Helpers are good in general, but this would need its own helper at the very least and I still think it would break the plugin functionality.